Update WebView2 docs for Ignite (#911)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/enterprise.md

---
description: Understand how to manage WebView2 applications
title: Managing WebView2 applications
author: MSEdgeTeam
ms.author: msedgedevrel
ms.date: 09/15/2020
ms.topic: conceptual
ms.prod: microsoft-edge
ms.technology: webview
keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html, enterprise, group policy, manageability 
---

# Managing WebView2 applications  

> [WebView2][WebView2Landing] is a component that developers use to build their applications, and the developers may deploy a [self-updating Evergreen WebView2 Runtime][Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview] on user device to power their applications. This document discusses how IT admins may manage WebView2 applications and Runtime. Feedback from both IT admins and developers are welcomed on [WebView2 Feedback repo][GithubMicrosoftedgeWebviewfeddback].  

Reviewer: ```suggestion
[WebView2][WebView2Landing] applications may depend on the [self-updating Evergreen WebView2 Runtime][Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview], which means the runtime can be installed on user devices. This document discusses how IT admins may manage WebView2 applications and the runtime. Feedback from both IT admins and developers is welcome on the [WebView2 Feedback repo][GithubMicrosoftedgeWebviewfeddback]. 
```

Author: Would probably leave the 1st couple of sentences as-is. I'd like to start the the doc describing what WV2/WVRT is and how they get onto the devices maintained by admins. 

Will fix the last sentence.

> ## Group policies for WebView2

Reviewer: ```suggestion
## Group policies for WebView2 applications
```

Author: Would leave it as WV2. This is not describing the apps' policies but WV2's policies

> IT admins may use group policy objects (GPO) to configure policy settings for WebView2. The following set of policies are/are not applicable to WebView2,

Reviewer: ```suggestion
IT admins may use Group Policy Objects (GPO) to configure policy settings for WebView2. The following list describes any policy differences that apply to WebView2.
```

Author: Will leave as-is. I'd rather start this as what policies WV2 has, rather than how it is different from the browser.

> * [Microsoft Edge - Update policies][EdgeUpdatePolicies] are available for IT admins to manage the install and update aspects of WebView2 Runtime. Microsoft Edge browser and WebView2 Runtime are updated through the same update mechanism. Unless a policy is channel-specific (e.g. `Update`), it applies to both the browser and WebView2 Runtime. e.g. `UpdateSuppressed` allows IT admins to set time during each day to suppress auto-update for both the browser and WebView2 Runtime. This enables IT admins to configure preferences and proxies once for browser and WebView2 Runtime to control their network bandwidth/traffic or for other purposes. IT admins may follow [Microsoft Edge's guide][ConfigureMicrosoftEdge] to configure Microsoft Edge - Update policies.

Reviewer: ```suggestion
* [Microsoft Edge - Update policies][EdgeUpdatePolicies] are available for IT admins to manage the install and update aspects of the WebView2 Runtime. The Microsoft Edge browser and WebView2 Runtime are updated using the same update mechanism. Unless a policy, such as `Update`, is channel-specific, it applies to both the browser and WebView2 Runtime. For example, `UpdateSuppressed` allows IT admins to set time during each day to suppress auto-update for both the browser and WebView2 Runtime. This policy enables IT admins to configure preferences and proxies for both the browser and WebView2 Runtime to control their network bandwidth. IT admins may follow [Configure Microsoft Edge policy settings on Windows][ConfigureMicrosoftEdge] to configure the update policies.
```

Reviewer: Is the following sentence necessary in the above paragraph?

This policy enables IT admins to configure preferences and proxies for both the browser and WebView2 Runtime to control their network bandwidth?

Author: Love the edits. Will apply them.

> This policy enables IT admins to configure preferences and proxies for both the browser and WebView2 Runtime to control their network bandwidth?

Yes, this explains (and gives a good example on) why we have one set of update policies apply to both edge and WV2, rather than have them separate.

> * [Microsoft Edge - Browser policies][EdgeBrowserPolicies] are disconnected from WebView2. This means the browser policies IT admins configure for Microsoft Edge browser do not apply to WebView2 applications. This is by design due to the fact that apps and browsers are often distinct use cases and that IT admins may not be aware of what applications use WebView2. Applying browser policies on WebView2 may have unintended consequences, e.g. IT admins disable JavaScript in the browser and all WebView2 apps using JavaScript are broken.  

Reviewer: ```suggestion
* [Microsoft Edge - Browser policies][EdgeBrowserPolicies] doesn't apply to WebView2 applications. This is by design because apps and browsers have different use cases, and IT admins may not be aware of what applications use WebView2. Applying browser policies on WebView2 may have unintended consequences. For example, IT admins may restrict JavaScript in the browser and all WebView2 apps using JavaScript are broken.  
```

Author: Applied edits. Switch out 'restrict' for 'block' since restrict is pretty vague.

> * (Coming soon) WebView2-specific policies – WebView2 will expose a small additional set of group policies in cases where managing WebView2 directly makes sense. Generally speaking, app developers are encouraged to build and expose their own group policies controlling their use of WebView2, as it is more straightforward for IT admins to manage the app rather than WebView2 directly.  

Reviewer: ```suggestion
* (Coming soon) WebView2-specific policies – WebView2 will expose additional group policies to manage WebView2 directly. We recommend app developers implement their own group policies to manage their app instead of controlling WebView2.  
```

Author: Will leave the first sentence as-is (I want to emphasize we're only going to have s SMALL set of policies in limited cases where it makes sense to manage WV directly) but shorten the 2nd sentence.
